src/FlaUI.WebDriver/Keys.cs

```diff
@@ -0,0 +1,505 @@
+        { '\uE012', "ArrowLeft" },
+        { '\uE014', "ArrowRight" },
+        { '\uE013', "ArrowUp" },
+        { '\uE00C', "Escape" },
+        { '\uE031', "F1" },
+        { '\uE032', "F2" },
+        { '\uE033', "F3" },
+        { '\uE034', "F4" },
+        { '\uE035', "F5" },
+        { '\uE036', "F6" },
+        { '\uE037', "F7" },
+        { '\uE038', "F8" },
+        { '\uE039', "F9" },
+        { '\uE03A', "F10" },
+        { '\uE03B', "F11" },
+        { '\uE03C', "F12" },
+        { '\uE019', "NumpadEqual" },
+        { '\uE01A', "Numpad0" },
+        { '\uE01B', "Numpad1" },
+        { '\uE01C', "Numpad2" },
+        { '\uE01D', "Numpad3" },
+        { '\uE01E', "Numpad4" },
+        { '\uE01F', "Numpad5" },
+        { '\uE020', "Numpad6" },
+        { '\uE021', "Numpad7" },
+        { '\uE022', "Numpad8" },
+        { '\uE023', "Numpad9" },
+        { '\uE025', "NumpadAdd" },
+        { '\uE026', "NumpadComma" },
+        { '\uE028', "NumpadDecimal" },
+        { '\uE029', "NumpadDivide" },
+        { '\uE007', "NumpadEnter" },
+        { '\uE024', "NumpadMultiply" },
+        { '\uE027', "NumpadSubtract" },
+    };
+
+    private static readonly Dictionary<char, string> s_shiftedKeyToCode = new()
+    {
+        { '~', "Backquote" },
+        { '|', "Backslash" },
+        { '{', "BracketLeft" },
+        { '}', "BracketRight" },
+        { '<', "Comma" },
+        { ')', "Digit0" },
+        { '!', "Digit1" },
+        { '@', "Digit2" },
+        { '#', "Digit3" },
+        { '$', "Digit4" },
+        { '%', "Digit5" },
+        { '^', "Digit6" },
+        { '&', "Digit7" },
+        { '*', "Digit8" },
+        { '(', "Digit9" },
+        { '+', "Equal" },
+        { '>', "IntlBackslash" },
+        { 'A', "KeyA" },
+        { 'B', "KeyB" },
+        { 'C', "KeyC" },
+        { 'D', "KeyD" },
+        { 'E', "KeyE" },
+        { 'F', "KeyF" },
+        { 'G', "KeyG" },
+        { 'H', "KeyH" },
+        { 'I', "KeyI" },
+        { 'J', "KeyJ" },
+        { 'K', "KeyK" },
+        { 'L', "KeyL" },
+        { 'M', "KeyM" },
+        { 'N', "KeyN" },
+        { 'O', "KeyO" },
+        { 'P', "KeyP" },
+        { 'Q', "KeyQ" },
+        { 'R', "KeyR" },
+        { 'S', "KeyS" },
+        { 'T', "KeyT" },
+        { 'U', "KeyU" },
+        { 'V', "KeyV" },
+        { 'W', "KeyW" },
+        { 'X', "KeyX" },
+        { 'Y', "KeyY" },
+        { 'Z', "KeyZ" },
+        { '_', "Minus" },
+        { '.', "Period" },
+        { '"', "Quote" },
+        { ':', "Semicolon" },
+        { '?', "Slash" },
+        { '\uE00D', "Space" },
+        { '\uE05C', "Numpad0" },
+        { '\uE056', "Numpad1" },
+        { '\uE05B', "Numpad2" },
+        { '\uE055', "Numpad3" },
+        { '\uE058', "Numpad4" },
+        { '\uE05A', "Numpad6" },
+        { '\uE057', "Numpad7" },
+        { '\uE059', "Numpad8" },
+        { '\uE054', "Numpad9" },
+        { '\uE05D', "NumpadDecimal" },
+    };
+
+    public static readonly char Null = '\uE000';
```

Comment:
Only `Null` and `LeftShift` are actually needed from these constants, but I added them all anyway.

Reply:
Actually, these should probably be `const` rather than `static readonly`.